api/path_node.go

```diff
@@ -116,12 +118,12 @@ func CreateHorizonDevice(device *HorizonDevice,
 
 	// There is no existing device registration in the database, so proceed to verifying the input device object.
 	if device.Id == nil || *device.Id == "" {
-		device_id := os.Getenv("HZN_DEVICE_ID")
+		device_id := cliutils.GetDeviceId()
 		if device_id == "" {
-			return errorhandler(NewAPIUserInputError("Either setup HZN_DEVICE_ID environmental variable or specify device.id.", "device.id")), nil, nil
+			return errorhandler(NewAPIUserInputError("Either setup HZN_DEVICE_ID or HZN_NODE_ID environmental variables or specify device.id.", "device.id")), nil, nil
```

Comment:
Can just mention `HZN_NODE_ID` because `HZN_DEVICE_ID` will be deprecated.

Reply:
addressed in recent changes